### PR TITLE
osu!tourney/ko.md

### PR DESCRIPTION
**Pull request by CookieCoMa**
osu!tourney/ko.md

### CODE REVIEW

**Comment by sr229** [comment]
Summary :
Formatting : Follows ASG Coverage 
Grammar : Skipped (Limitations)

You're in the right track

File: wiki/osu!tourney/kr.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+**토너먼트 프로그램 사용시 스트리밍을 하는 대회, 멀티유저 이벤트 이외의 클라이언트 사용을 금지합니다. 또한 클라이언트는 개인용으로 사용할 수 없습니다.**
+
+사용의 **문제**가 있는 경우  **[이 스레드](https://osu.ppy.sh/forum/t/327120)**에서 **[문제해결](Osu!tourney#Troubleshooting "wikilink")** 섹션을 읽거나, 게시 할 수 있습니다. 
+
+osu!tourney 무엇인가?
+===================
+
+osu!tourney 클라이언트는 멀티방에서 모든 플레어어의 플레이를 동시에 관전하는데 사용되는 공식 스트리밍 클라이언트입니다. 클라이언트는 월드컵용으로 제작되었기 때문에 약간의 제약이 있을 수도 있습니다.
+
+토너먼트 클라이언트는 양팀을 합쳐서 16명 까지 지원 가능하며 각 팀 인원은 최소 1명에서 최대 8명 까지 팀을 구성할 수 있습니다. The number of windows opened cannot currently be changed, but this behaviour will be modified in the future.
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Not translated: `The number of windows opened cannot currently be changed, but this behaviour will be modified in the future.`

File: wiki/osu!tourney/kr.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+-   **Height:** osu!tourney는 자동으로 16:9로 계산되도록 폭을 계산합니다. 검은 색 컨트롤(토너먼트 자제제어판) 패널에는 해당 높이가 포함되어 있지 않으므로 1080p/s의 스트리밍 하기 위해 1440p인치 화면이 필요합니다. 기본 값:720, 허용 범위:640x720 
+
+tournament.cfg 파일을 수정한후에는 osu!를 다시 시작하면 osu! tourney 클라이언트는 업데이트 된 설정으로 시작합니다.
+
+osu!tourney 사용법
+-----------------
+
+| 클라이언트/서버 클라이언트의 기본 인터페이스 |
+| ----------------------------------------- |
+| ![osu!tourney Interface](Osutourneymain.png "osu!tourney Interface") |
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+
+위 사진은 클라이언트의 인터페이스입니다. 클라이언트 자체 사용은 매우 간단합니다. 화면 상단은 두 개의 파트로 나뉘여 있으며 멀티 방의 두 팀을 대표합니다. 각각의 osu! 화면은 멀티 룸의 슬롯에 해당됩니다. 이것은 각각의 플레이어들이 각자의 자리에 있을 것 이라는 것을 의미 합니다. 
+
+| 플레이어는 멀티방에 위치에 따라 윈도우창에 할당됩니다. |
+| ----------------------------------------------------------------------------------- |
+| ![osu!tourney Player Windows](Osutourneywindows.png "osu!tourney Player Windows")   |
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+| 플레이어는 멀티방에 위치에 따라 윈도우창에 할당됩니다. |
+| ----------------------------------------------------------------------------------- |
+| ![osu!tourney Player Windows](Osutourneywindows.png "osu!tourney Player Windows")   |  
+
+제대로된 진행을 위해서 선수들이 멀티 방에서 적절한 슬롯에 있는 지 확인 하십시오. 정확한방법에 대한 내용은 멀티플레이어 세션 생성 및 취급에서 확인 가능 합니다.
+
+| osu!tourney 컨트롤 패널 |
+| ----------------------------------------- |
+| ![osu!tourney control panel](Osutourneypanel.png "osu!tourney control panel") |
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+
+![When the multiplayer rooms were created correctly, they will be listed instead of the instructions](Osutourneyroomlist.png "When the multiplayer rooms were created correctly, they will be listed instead of the instructions")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+
+![osu!tourney assigns team names and usernames automatically](Osutourneyidle.png "osu!tourney assigns team names and usernames automatically")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+
+![osu!tourney does not simply spectate. It also shows the current team score and the currently played song](Osutourneyspectate.png "osu!tourney does not simply spectate. It also shows the current team score and the currently played song")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+
+![osu!tourney declares a winner automatically, adds a score and plays another sprite when showing the results screen. Failed players are blacked out.](Osutourneyresults.png "osu!tourney declares a winner automatically, adds a score and plays another sprite when showing the results screen. Failed players are blacked out.")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+
+클라이언트 하단에 제어판이 표시 됩니다.이것은 당신이 원하는 멀티방의 이름을 짓기 위해 필요한 템플릿을 보여드립니다. 그것은 tournament.cfg 파일설정과 2개의 팀 이름으로 정의된 약어로 구성됩니다. **[자세한 내용은 멀티 플레이어 세션 생성 및 취급을 참조하십시오.](Osu!tourney#Multiplayer_session_creation_and_handling "wikilink")**
+
+멀티 방이 적절하게 되어 있는 경우, 제어판은 적절하게 제작된 멀티 방을 나열합니다. 멀티 플레이어를 자동으로 선택할 수 있도록 선택할 수 있는 멀티 방을 클릭합니다. 모든 것을 제대로 설정하였다면 자동으로 클라이언트 사용자에게 타 플레이어들의 연주나 노래 소리를 듣거나 보게 됩니다.  
+
+경기가 종료되면 점수가 높은 팀에게 별 하나가 채워지며 팀 점수(별)를 증가됩니다. 또한 팀점수를 수동으로 변경할 수 있도록 하여. **마우스 오른쪽키와 왼쪽키를 클릭하여 별의 개수를 수동으로 변경 할 수 있습니다.**.
+
+![osu!tourney with activated annotation](Osutourneywarmup.png "osu!tourney with activated annotation")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text.

File: wiki/osu!tourney/kr.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+-   `!mp start [<time>]` - 설정된 시간 (초) 후에 또는 시간이 없는 경우에는 즉시 시작합니다.
+-   `!mp abort` - 경기를 중단합니다.
+-   `!mp kick` <username> - 플레이어를 추방시킵니다.
+-   `!mp team` <username> <colour> - 플레이어를 지정된 팀으로 이동시킵니다.
+
+    `colour` - red, blue
+
+-   `!mp map` <mapid> `[<playmode>]` - 방의 비트맵과 플레이모드를 변경합니다.
+
+    `playmode` - 0: osu!, 1: Taiko, 2: Catch The Beat, 3: osu!Mania
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]:
```Markdown
`playmode` - 0: osu!, 1: osu!taiko, 2: osu!catch, 3: osu!mania
```

File: wiki/osu!tourney/kr.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+`!mp set 0 2` - 개인전, 스코어 모드로 설정합니다.
+
+`!mp start` - 즉시 게임을 시작합니다.
+
+방 관리
+---------------
+
+osu! tourney 클라이언트가 방을 성공적으로 인식하려면 !mp move 및 !mp team 명령을 사용하여 방에 들어오는 플레이어에게 올바른 슬롯으로 배정해야합니다.
+
+**[Using osu!tourney](Osu!tourney#Using_osu!tourney "wikilink")** 설명한대로, 클라이언트의 각 창은 멀티 플레이어 로비슬롯에 배정됩니다. tournament.cfg의 TeamSize 변수 값에 따라 왼쪽 팀이 첫 번째 슬롯을 차지하고 오른쪽 팀이 아래슬롯을 차지합니다.
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text

File: wiki/osu!tourney/kr.md [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+
+| 각 멀티방 슬롯에 대한 osu!tourney에 대응 |
+| ------------------------------------------------------------------------------------- |
+| ![osu!tourney Player Assignment](Osutourneyassignment.png "osu!tourney Player Assignment") |
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text

File: wiki/osu!tourney/kr.md [hunk 13]
```diff
@@ -0,0 +1,353 @@
...
+| ![TeamSize = 2](Teamsize2.png "TeamSize = 2")   | 
+
+| TeamSize = 1 |
+| -----------------------|
+| ![TeamSize = 1](Teamsize1.png "TeamSize = 1")   | 
+
+스킨
+--------
+
+![osu!tourney can be customized in various ways](Osutourneycustom.png "osu!tourney can be customized in various ways")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text

File: wiki/osu!tourney/kr.md [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+폴더의 경로는 `.../osu!/Skins/User/tournament`입니다.스킨 요소는 이 폴더(tournament)에 배치 할 수 있으며 `*.jpg`, `*.png` and `*.mp4` 파일 확장자를 지원합니다. 적절한 스킨 배치를 위하여 다음요소에 따라 이름을 지정하십시오.
+
+-   `background` - 이것은 전반적인 시간동안 osu!tourney에 사용되는 뒷 배경입니다.
+-   `background-win1` (optional) - 배경색은 어둡게 됩니다. 왼쪽 팀이 이겼을 때 결과화면에서 이 사진으로 사라집니다. 결과 화면을 떠날 때 백그라운드로 다시 표시됩니다. 이 사진이 배치되지 않을 경우 '배경'이 그대로 유지됩니다.
+-   `background-win2` (optional) - 배경색은 어둡게 됩니다. 오른쪽 팀이 이겼을 때 결과화면에서 이 사진으로 사라집니다. 결과 화면을 떠날 때 백그라운드로 다시 표시됩니다. 이 사진이 배치되지 않을 경우 '배경'이 그대로 유지됩니다.
+-   `tourney-title` (optional) - 이 영상은 배경 화면 맨 위와 아래쪽에 위치합니다. 이는 로고의 로고를 고정하는 데 사용할 수 있습니다(이것은 영상으로만 작동합니다.)
+
+이것에 최적화된 해상도는 1280x720 60FPS입니다.
+
+![osu!tourney with team icons. The left icon is named Team 1.png and the right icon is named Team 2.png](Osutourneyicons.png "fig:osu!tourney with team icons. The left icon is named Team 1.png and the right icon is named Team 2.png")
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Translate the text

File: wiki/osu!tourney/kr.md [hunk 15]
```diff
@@ -0,0 +1,353 @@
...
+-   정확한 멀티방 이름을 사용했는지 확인하십시오. (see **[매칭 설정](Osu!tourney#Match_creation "wikilink")**). 참조하십시오.
+-   실수한 경우 방을 다시 만듭니다.
+
+<!-- -->
+
+**j) Twitch.tv와 같은 서비스에 내 토너먼트를 스트리밍하려면 어떻게 해야 합니까?**
+
+스트리밍을 설정하려면 Open Broadcaster Software를 사용하는 것이 좋지만 액세스 권한이 있는 다른 스트리밍 클라이언트를 자유롭게 사용할 수도 있습니다. osu! tourney의 윗부분의 해상도는 1280x720입니다. 검은 색 컨트롤 패널을 스트리밍하지 마십시오.
+
+<!-- -->
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: Replace All `<!-- -->` with empty space. One of the unfixed errors in en.md during conversion.

File: wiki/osu!tourney/kr.md (hunk 10 above)
Comment (RaikireHiuduo) [inline wiki/osu!tourney/kr.md]: New naming convention used. The one provided was based on the old name version.

Just replace the 

```Markdown
`playmode` - 0: osu!, 1: Taiko, 2: Catch The Beat, 3: osu!Mania
```` 

with the line provided below:

```Markdown
`playmode` - 0: osu!, 1: osu!taiko, 2: osu!catch, 3: osu!mania
```

No need to translate.

Comment (CookieCoMa) [inline wiki/osu!tourney/kr.md]: Thank you. Fixed

**Comment by CookieCoMa** [comment]
Fixed, Sorry for making a simple mistake.

**Comment by RaikireHiuduo** [comment]
Context: Check #149 . https://github.com/ppy/osu-wiki/pull/149/files

**Comment by CookieCoMa** [comment]
All Fixed Thank you

File: wiki/osu!tourney/ko.md [hunk 16]
```diff
@@ -0,0 +1,344 @@
...
+
+### 스킨
+--------
+
+![osu!tourney 다양한 방법으로 커스텀마이즈 할 수 있다.](Osutourneycustom.png "osu!tourney can be customized in various ways")
+
+osu! tourney는 다양한 디자인을 위해 클라이언트 스킨이 수정 가능 합니다. 
+그것을 위해서 당신은 토너먼트 폴더에 스킨을 넣어야 합니다.
+
+폴더의 경로는 `.../osu!/Skins/User/tournament`입니다.스킨 요소는 이 폴더(tournament)에 배치 할 수 있으며 `*.jpg`, `*.png` and `*.mp4` 파일 확장자를 지원합니다. 적절한 스킨 배치를 위하여 다음요소에 따라 이름을 지정하십시오.
```
Comment (MegaApplePi) [inline wiki/osu!tourney/ko.md]: all references to mp4 and/or video are to be removed (osu!tourney doesn't use them anymore; see #145)

**Comment by CookieCoMa** [comment]
Fixed!, Thank you all the time.

**Comment by CookieCoMa** [comment]
I've fixed it, but I do not know how to change it properly. Can you check it?

File: wiki/osu!tourney/ko.md [hunk 17]
```diff
@@ -0,0 +1,350 @@
...
+이 사진이 배치되지 않을 경우 '배경'이 그대로 유지됩니다.
+-   `background-win2` (optional) - 배경색은 어둡게 됩니다. 오른쪽 팀이 이겼을 때 결과화면에서 이 사진으로 사라집니다. 
+결과 화면을 떠날 때 백그라운드로 다시 표시됩니다. 
+이 사진이 배치되지 않을 경우 '배경'이 그대로 유지됩니다.
+-   `tourney-title` (optional) - 이 영상은 배경 화면 맨 위와 아래쪽에 위치합니다. 
+이는 로고의 로고를 고정하는 데 사용할 수 있습니다(이것은 영상으로만 작동합니다.)
+
+이것에 최적화된 해상도는 1280x720 60FPS입니다.
+
+![osu!tourney 팀 아이콘 입니다. 왼쪽 아이콘 이름은 Team 1.png 그리고 오른쪽 아이콘 이른은 named Team 2.png 입니다.](Osutourneyicons.png "fig:osu!tourney with team icons. The left icon is named Team 1.png and the right icon is named Team 2.png")
```
Comment (MegaApplePi) [inline wiki/osu!tourney/ko.md]: this `"fig:osu!tourney with team icons. The left icon is named Team 1.png and the right icon is named Team 2.png"`  can be translated. Though, remove the `fig:` part - it isn't needed and is most likely leftover from the wiki transition.

File: wiki/osu!tourney/ko.md [hunk 18]
```diff
@@ -0,0 +1,350 @@
...
+
+(예: 국기 또는 사용자 아바타)을 skin폴더에 생성할 수 있습니다. 아이콘은 반드시 /osu!/Skins/User/tournament/icons  `.../osu!/Skins/User/tournament/icons`. 이 경로로 배치해야 합니다. 또한 모든 아이콘은 팀 이름과 정확히 일치해야 합니다.
+
+만약 당신의 경기가 Test Tourney: '(Team 1) vs (Team 2)' 라 불린다면 당신은 그 각각의 팀의 아이콘들은 반드시'(team 1)과 (team 2)' 불리어야 합니다. 
+또한 아이콘은 `*.jpg`또는 `*.gif`형식으로 되어 있으며, '50x50 최적 해상도를 가지고 있어야 합니다.
+
+**[다운로드 template.psd 여기] (http://osu.ppy.sh/images/tournament/template.zip)** 를 통해 개인 스킨을 간편하게 생성할 수 있습니다.
```
Comment (MegaApplePi) [inline wiki/osu!tourney/ko.md]: `**[다운로드 template.psd 여기](http://osu.ppy.sh/images/tournament/template.zip)**` (remove the space between the `](` part

File: wiki/osu!tourney/ko.md [hunk 19]
```diff
@@ -72,7 +72,7 @@ osu!를 닫은 후 다른 옵션을 위하여 위의 파일을 편집하거나 
 -   **TeamSize:** 팀의 크기를 지정합니다. 이에 따라 osu!tourney는 1-8개의 창을 엽니다(최대 값은 8입니다.)
 -   **ClientNameSize:** 클라이언트 속 플레이어의 이름 크기를 변경합니다.
 -   **privateserver:** 절대손대지마세요. 이것은 당신들에게 중요하지 않습니다.
--   **acronym:** 당신의 토너먼트에 대한 약자를 넣으세요, 예) 'OWC 2013'. **[멀티플레이 세션 생성 및 조작에 대한 자세한 내용](Osu!tourney#플레이어 세션 생성 및 조작 "wikilink")**.
+-   **acronym:** 당신의 토너먼트에 대한 약자를 넣으세요, 예) 'OWC 2013'. **[멀티플레이 세션 생성 및 조작에 대한 자세한 내용](#플레이어 세션 생성 및 조작)**.
```
Comment (MegaApplePi) [inline wiki/osu!tourney/ko.md]: `**[멀티플레이 세션 생성 및 조작에 대한 자세한 내용](#플레이어-세션-생성-및-조작)**`

File: wiki/osu!tourney/ko.md [hunk 20]
```diff
@@ -111,7 +111,7 @@ _osu!tourney 컨트롤 패널_
 
 클라이언트 하단에 제어판이 표시됩니다.이것은 당신이 원하는 멀티방의 이름을 짓기 위해 필요한 템플릿을 보여드립니다. 
 그것은 tournament.cfg 파일설정과 2개의 팀 이름으로 정의된 약어로 구성됩니다.
-**[자세한 내용은 멀티플레이어 세션 생성 및 조작을 참조하십시오.](Osu!tourney#멀티플레이어 세션 생성 및 조작 "wikilink")**
+**[자세한 내용은 멀티플레이어 세션 생성 및 조작을 참조하십시오.](#멀티플레이어 세션 생성 및 조작)**
```
Comment (MegaApplePi) [inline wiki/osu!tourney/ko.md]: this link doesn't work (I have fixed it but the link doesn't do anything -- meaning: the section heading name and this link don't match)

~~`**[자세한 내용은 멀티플레이어 세션 생성 및 조작을 참조하십시오.](#멀티플레이어-세션-생성-및-조작)**`~~

File: wiki/osu!tourney/ko.md [hunk 21]
```diff
@@ -150,7 +150,7 @@ osu!tourney을 사용하여 멀티 방을 원활하게 상호작용을 하기 
 `Your_acronym_in_tournament.cfg: (팀 이름 1) vs (팀 이름 2)`
 
 `Your_acronym_in_tournament.cfg` 설치 중에 'tournaments.cfg'에서 정의한 값으로 대체됩니다. 
-예를 들어, 이 값은 **[설정]**(Osu!tourney#설정 "wikilink"). "Test Tourney"로 정의되었습니다. 다음 예제에서는 템플릿을 사용합니다.
+예를 들어, 이 값은 **[설정]**(#설정). "Test Tourney"로 정의되었습니다. 다음 예제에서는 템플릿을 사용합니다.
```
Comment (MegaApplePi) [inline wiki/osu!tourney/ko.md]: `**[설정](#설정)**` -- move the asterisks

**Pull request merged**